community-plugins.json

```diff
@@ -4668,5 +4668,13 @@
         "description": "Create folder of snippets to activate them in one click !",
         "repo": "Mara-Li/obsidian-group-snippets",
         "branch": "master"
+    },
+    {
+        "id": "obsidian-markdown-file-suffix",
```

Comment:
I suggest removing the `obsidian-` prefix. It's not needed. Also the name and ID should probably better match the use of the plugin. Maybe just `mdx-svx-extension-support`.